src/index.html

```diff
@@ -2,13 +2,48 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navbar">
+      <a href="#home">
+        <img src="./images/logo.png" alt="logo"
+        class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones </a>
+          </li>
+          <li class="nav__item" data-qa="hover">
+            <a href="#lapancomp" class="nav__link">Laptops &Computers </a>
```

Comment:
Try to change it and check your tests
```suggestion
            <a href="#lapancomp" class="nav__link">Laptops & Computers </a>
```

Also check your image position. It differs from the original according to tests